spec/queries/project/subscriptions_pledged_value_query_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Project::SubscriptionsPledgedValueQuery do
+  describe ".call" do
+    let(:subscriptions_pledged_value) { Project::SubscriptionsPledgedValueQuery.new(project) }
```

Comment:
I'm not really sure if makes sense if I call the class method directly, because it is more like an alias for the instance method.